reloc.ml

```diff
@@ -1203,7 +1203,11 @@ let normalize_path path =
   let rec loop acc path =
     match path with
     | "."  :: path -> loop acc path
-    | ".." :: path -> loop (List.tl acc) path
+    | ".." :: path -> begin
+        match acc with
+        | _ :: q -> loop q path
+        | [] -> loop [".."] path
```

Comment:
This does prevent the crash, but it doesn't always return the right result - e.g. `../..` would normalise to `[]` instead of `[".."; ".."]`. I think this tweak fixes it:
```suggestion
        | previous :: acc when previous <> ".." -> loop acc path
        | _ -> loop (".." :: acc) path
```

Reply:
Duh 🤕 . Thanks for catching this!